.github/workflows/e2e.yaml

```diff
@@ -84,6 +84,28 @@ jobs:
           echo "========== git verify-commit =========="
           git verify-commit HEAD
 
+          echo "========== gitsign verify =========="
+          gitsign verify \
+            --certificate-github-workflow-repository=${{ github.repository }} \
+            --certificate-github-workflow-sha=${{ github.sha }} \
+            --certificate-oidc-issuer="https://token.actions.githubusercontent.com" \
+            --certificate-identity="https://github.com/${{ github.workflow_ref }}"
+
+          # Extra debug info
+          git cat-file commit HEAD | sed -n '/BEGIN/, /END/p' | sed 's/^ //g' | sed 's/gpgsig //g' | sed 's/SIGNED MESSAGE/PKCS7/g' | openssl pkcs7 -print -print_certs -text
+      - name: Test Sign and Verify commit - offline verification
+        env:
+          GITSIGN_REKOR_MODE: "offline"
```

Comment:
Hm, is there any way to double check that validation actually happens offline? If you typo'd `GITSIGN_REKOR_MODE` this test wouldn't check the right thing.

You could turn off network access for the `gitsign verify` commands
https://github.com/orgs/community/discussions/24975